### PR TITLE
Update Ember Core Concepts image to use Octane idioms

### PR DESCRIPTION
**Pull request by muziejus**
This is the best I could get to pushing my changes to @amyrlam's branch on #1460.

### CODE REVIEW

**Comment by ijlee2** [comment]
Hi, @muziejus. It looked like Amy and Godfrey may have context to review your PR. Let me add them as reviewers, as well as the Learning Team (in case there's someone else who can help review).

File: public/images/ember-core-concepts/ember-core-concepts.svg [hunk 1]
```diff
@@ -1,25 +1,25 @@
+<<<<<<< Updated upstream
```
Comment (amyrlam) [inline public/images/ember-core-concepts/ember-core-concepts.svg:1]: @muziejus Thank you for making the new PR! Was on vacation for a few days. 

I think that this `ember-core-concepts.svg` file had a bad rebase or something. Is it possible to perhaps delete the existing file and upload the new one from the external program? (Am not sure what that program is, Adobe Illustrator?)

When looking at it locally, I'm having trouble figuring out what to do in the rebase (line 1, 133, 244, 247, 249 etc). 

BTW if it's easier to discuss on Discord, I'm @ amyrlam on there.

Comment (muziejus) [reply public/images/ember-core-concepts/ember-core-concepts.svg:1]: @amyrlam do you mean I just send you my files outside of git?

Comment (muziejus) [reply public/images/ember-core-concepts/ember-core-concepts.svg:1]: (also I'm @muziejus on the ember discord)

**Pull request closed without merging**